pygsti/report/workspaceplots.py

```diff
@@ -3833,10 +3833,10 @@ def _create(self, projections,
             new_lmargin = fig.plotlyfig.layout.margin.l - 20
 
             #add annotations for the upper and lower triangles.
-            fig.plotlyfig.add_annotation(x= .92, y= -.16, xref= 'paper', yref= 'paper',
+            fig.plotlyfig.add_annotation(x= 0.92, y= -.16, xref= 'paper', yref= 'paper',
```

Comment:
Missing the ``y=-0.16``; re-run the grep (or whatever else) to include strings of the form ``-.{0-9}{0-9}*``